README.md

```diff
@@ -22,8 +22,25 @@ A destination connector pushes data from upstream resources to an external resou
 |----------------------|-------------------------------------------------------------------------------------------------------------------------------------|----------|---------------|
 | `endpoint`           | Host of the Weaviate instance.                                                                                                      | true     | ""            |
 | `scheme`             | Scheme of the Weaviate instance. Values: https, http.                                                                               | false    | "https"       |
-| `apiKey`             | A Weaviate API key.                                                                                                                 | false    | ""            |
 | `class`              | The class name as defined in the schema. A record will be saved under this class unless it has the `weaviate.class` metadata field. | true     | ""            |
 | `moduleHeader.name`  | Name of the header configuring a module (e.g. `X-OpenAI-Api-Key`).                                                                  | false    | ""            |
 | `moduleHeader.value` | API key for the module defined above.                                                                                               | false    | ""            |
 | `generateUUID`       | Generate a UUID for records (an MD5 sum of a record's key).                                                                         | false    | "false"       |
```

Comment:
new auth configs are not added to the table, and `apiKey` is removed?

Reply:
I've split the docs into two parts. The first one is here and then below there's a section dedicated to auth. only. The table columns grew a bit too big.

Reply:
Sure, I can move it back into the table! However, at a later point in time, we can maybe discuss how to improve the format, since this isn't the first time at least I come across this.

Reply:
sgtm!

Reply:
Done!